src/FakeItEasy/Creation/FakeAndDummyManager.cs

```diff
@@ -5,19 +5,15 @@ namespace FakeItEasy.Creation
     using System.Reflection;
     using FakeItEasy.Core;
 
-    /// <summary>
-    /// The default implementation of the IFakeAndDummyManager interface.
-    /// </summary>
-    internal class DefaultFakeAndDummyManager
-        : IFakeAndDummyManager
+    internal class FakeAndDummyManager
     {
         private static readonly ConcurrentDictionary<Type, Func<ProxyOptions, IFakeOptions>> FakeOptionsFactoryCache = new ConcurrentDictionary<Type, Func<ProxyOptions, IFakeOptions>>();
 
         private readonly IFakeObjectCreator fakeCreator;
         private readonly ImplicitOptionsBuilderCatalogue implicitOptionsBuilderCatalogue;
         private readonly IDummyValueResolver dummyValueResolver;
 
-        public DefaultFakeAndDummyManager(IDummyValueResolver dummyValueResolver, IFakeObjectCreator fakeCreator, ImplicitOptionsBuilderCatalogue implicitOptionsBuilderCatalogue)
+        public FakeAndDummyManager(IDummyValueResolver dummyValueResolver, IFakeObjectCreator fakeCreator, ImplicitOptionsBuilderCatalogue implicitOptionsBuilderCatalogue)

```

Comment:
We weren't taking advantage of the interface, and I didn't feel like adding a method to _it_ and the default implementation!

Reply:
The new `CreateFake` overload right downstairs. As you say, eliding the interface isn't strictly necessary here. I can move to a separate PR if you like.

Reply:
Ah, yes. No, you can leave it there, it's fine